### PR TITLE
Icon Harmonization

### PR DESCRIPTION
**Pull request by Michadelic**
* Iconography adjusted to use icons the SAPUI5 icon font
* Use a variant of the official ui5 newscast icon
* Use the official ui5-schemas logo
* Use demo kit visualization provided by UX

* documentation: demo kit, courses, demos, tutorials
* releases: npm, roadmap
* projects: ui5-schemas
* community: ui5 newscast

### CODE REVIEW

File: src/components/icon/styles.css [hunk 1]
```diff
@@ -93,7 +93,7 @@
 	background-image: url("./img/ui5-lab.svg");
 	}
 .icon.name_ui5_schemas {
-	background-image: url("./img/source-code.svg");
+	background-image: url("./img/ui5-schemas.svg");
```
Comment (avh-sap) [inline src/components/icon/styles.css]: "ui5-schemas.svg" seems to be missing

Comment (Michadelic) [reply src/components/icon/styles.css]: thx, i forgot to add it

**Comment by Michadelic** [comment]
Ok i now compressed the SVGs as requested...

As we repeatedly discovered undocumented guidelines and best practices for this project during the code review, suggestion to add a GUIDELINES.md markdown document for contributors and link it from the CONTRIBUTING.md file.

File: src/components/icon/styles.css [hunk 2]
```diff
@@ -93,7 +93,7 @@
 	background-image: url("./img/ui5-lab.svg");
 	}
 .icon.name_ui5_schemas {
-	background-image: url("./img/source-code.svg");
```
Comment (pavelkornev) [inline src/components/icon/styles.css]: Please remove this unused icon from the project.

File: src/components/icon/img/ui5-newscast.svg [hunk 3]
```diff
@@ -1 +1 @@
...
... 8.11 0 0 1-3.42 5 7.88 7.88 0 0 1-3 1.16l-3.93.7z"/><path class="D" d="M50 7.77l.42 2.32a9.19 9.19 0 0 1 0 2.85L27.8 17a8.81 8.81 0 0 1-.8-2l-.58-3.16L50 7.6z"/><path class="E" d="M27.8 17l22.7-4.06s-.54 6.46-9 7l-5.74 1A7.79 7.79 0 0 1 27.79 17z"/></g><g clip-path="url(#B)"><path class="C" d="M39.43 18.43a7.94 7.94 0 0 1-3.15-.07 8 8 0 0 1-5-3.46 7.75 7.75 0 0 1-1.12-2.9l-1.4-7.86L52.32-.1l1.4 7.86a7.72 7.72 0 0 1-.07 3.14 8.11 8.11 0 0 1-3.42 5 7.88 7.88 0 0 1-3 1.16l-3.93.7z"/><path class="D" d="M53.3 5.3l.42 2.32a9.19 9.19 0 0 1 0 2.85l-22.65 4.06a8.81 8.81 0 0 1-.8-2l-.57-3.16 23.57-4.23z"/><path class="E" d="M31.07 14.54l22.7-4.06s-.54 6.46-9 7l-5.74 1a7.77 7.77 0 0 1-7.95-3.94z"/></g><g clip-path="url(#C)"><path class="C" d="M33.23,23.53a8,8,0,0,1-3.15-.07,8.13,8.13,0,0,1-2.82-1.24A8.34,8.34,0,0,1,25.12,20,7.85,7.85,0,0,1,24,17.08L22.56,9.22,46.13,5l1.4,7.86A7.77,7.77,0,0,1,47.46,16a7.92,7.92,0,0,1-1.23,2.82A8.22,8.22,0,0,1,44,21a7.65,7.65,0,0,1-3,1.16l-3.92.7Z"/><path class="D" d="M47.1 10.4l.4 2.33a8.78 8.78 0 0 1 0 2.84l-22.65 4.06a9 9 0 0 1-.79-2l-.58-3.16 23.58-4.22z"/><path class="E" d="M24.87 19.64l22.7-4.06s-.53 6.46-9 7l-5.74 1a7.8 7.8 0 0 1-7.95-3.94z"/></g></svg>
```
Comment (pavelkornev) [inline src/components/icon/img/ui5-newscast.svg]: Though, the viewport of this icon is fine, but fractional pixels inside are overlapping and produce an unpleasant noise (hardness) effect. This has be to fixed by the designer:

File: src/components/documentation/icon.svg [hunk 4]
```diff
@@ -1 +1 @@
...
... class="E" d="M15.75 175.72a.35.35 0 0 1 .07.47l-5.53 8.12a.36.36 0 0 1-.29.23.39.39 0 0 1-.35-.13l-3.76-4.16a.31.31 0 0 1-.12-.27.45.45 0 0 1 .14-.27l.62-.63a.36.36 0 0 1 .58 0l2.4 2.4a.39.39 0 0 0 .32.11.36.36 0 0 0 .3-.21l4.36-6.2a.4.4 0 0 1 .29-.2.45.45 0 0 1 .33.1z"/><path class="B" d="M24.26 192.2h75.32v9.04H24.26z"/><path class="E" d="M15.75 192.2a.36.36 0 0 1 .07.48l-5.53 8.1a.35.35 0 0 1-.29.22.38.38 0 0 1-.35-.12l-3.76-4.1a.33.33 0 0 1-.12-.28.45.45 0 0 1 .14-.27l.62-.62a.36.36 0 0 1 .58 0l2.4 2.4a.39.39 0 0 0 .32.11.36.36 0 0 0 .3-.21l4.36-6.2a.36.36 0 0 1 .29-.2.38.38 0 0 1 .33.1z"/><path class="B" d="M24.26 207.6h75.32v9.04H24.26z"/><path class="E" d="M15.75 207.6a.35.35 0 0 1 .07.47l-5.53 8.12a.35.35 0 0 1-.29.22.38.38 0 0 1-.35-.12l-3.76-4.16a.33.33 0 0 1-.12-.28.47.47 0 0 1 .14-.27l.62-.62a.36.36 0 0 1 .58 0l2.4 2.4a.39.39 0 0 0 .32.11.36.36 0 0 0 .3-.21l4.36-6.2a.36.36 0 0 1 .29-.2.38.38 0 0 1 .33.1z"/><path class="B" d="M5.8 152.27h93.92V159H5.8z"/><path class="E" d="M161.47 134.8h118.76v9.04H161.47z"/><path class="F" d="M160.4 175.72h95.24v9.04H160.4zm0 16.48h95.24v9.04H160.4zm0 15.4h95.24v9.04H160.4z"/><path class="E" d="M161.84 152.27h93.92V159h-93.92z"/></svg>
```
[the remark this answers is not in the document]
Comment (pavelkornev) [reply src/components/documentation/icon.svg]: `viewBox="0 0 304.12 240.11"`

File: src/components/tile/type_feature/3.svg [hunk 5]
```diff
@@ -1 +1 @@
...
...0,1,.13-.34l.64-.71a.41.41,0,0,1,.31-.14.42.42,0,0,1,.32.14l.28.31.2-.1a.66.66,0,0,1,.21-.08v-.43a.52.52,0,0,1,.12-.36.45.45,0,0,1,.33-.14h.89a.45.45,0,0,1,.33.14.52.52,0,0,1,.12.36v.43a.66.66,0,0,1,.21.08l.2.1.28-.31A.42.42,0,0,1,50,17a.43.43,0,0,1,.31.14l.64.71a.45.45,0,0,1,.13.34.51.51,0,0,1-.13.35l-.28.31.09.21c0,.07.05.15.08.23Zm0,.5h-.73l-.09-.34a1.46,1.46,0,0,0-.07-.19c0-.07,0-.14-.07-.2l-.16-.3.24-.27.28-.3L50,17.48l-.28.3-.24.25-.29-.15L49,17.79l-.17-.07-.31-.12v-.78h-.89v.78l-.31.12-.18.07a.91.91,0,0,0-.17.09l-.28.15-.24-.25-.28-.3-.63.69.28.3.22.27-.14.3c0,.06-.05.13-.08.2l-.06.19-.11.34h-.71v1h.71l.11.34.06.18c0,.06.05.12.08.19l.14.32L45.8,22l-.28.31.63.69.28-.31.24-.26.28.17.17.08.18.07.31.11v.8h.89v-.8l.31-.11.17-.07.17-.08.29-.17.24.26L50,23l.63-.69L50.31,22l-.24-.26.16-.32c0-.07.05-.13.07-.19a1.34,1.34,0,0,0,.07-.18l.09-.34h.73Zm-3.14-1a1.27,1.27,0,0,1,1,.43,1.52,1.52,0,0,1,.39,1.05,1.49,1.49,0,0,1-.39,1,1.28,1.28,0,0,1-1.91,0,1.49,1.49,0,0,1-.39-1,1.52,1.52,0,0,1,.39-1.05A1.26,1.26,0,0,1,48.05,18.78Zm0,2.46a.82.82,0,0,0,.63-.29,1,1,0,0,0,.27-.69,1,1,0,0,0-.27-.69.8.8,0,0,0-.63-.3.82.82,0,0,0-.63.3,1,1,0,0,0-.26.69,1,1,0,0,0,.26.69A.85.85,0,0,0,48.05,21.24Z"/></svg>
```
[the remark this answers is not in the document]
Comment (pavelkornev) [reply src/components/tile/type_feature/3.svg]: `viewBox="0 0 58.42 51.25"`

**Comment by pavelkornev** [comment]
@Michadelic with this PR the icon on the Documentation page (under the Courses section) is the same as the icon on the Community page for "UI5 Logo Guidelines" block. The same icons, but different actions...

**Pull request merged**